python/lsst/meas/astrom/astromLib.i

```diff
@@ -11,26 +11,61 @@ Python interface to lsst::meas::astrom
 %{
 #include "lsst/afw/image.h"
 #include "lsst/afw/cameraGeom.h"
+#include "lsst/afw/geom.h"
 #include "lsst/afw/table.h"
 #include "lsst/afw/image/Wcs.h"
 #include "lsst/afw/image/TanWcs.h"
 #include "lsst/meas/astrom/matchOptimisticB.h"
 #include "lsst/meas/astrom/makeMatchStatistics.h"
+#include "lsst/meas/astrom/PolynomialTransform.h"
+#include "lsst/meas/astrom/SipTransform.h"
+#include "lsst/meas/astrom/ScaledPolynomialTransformFitter.h"
 %}
 
 %include "lsst/p_lsstSwig.i"
 %initializeNumPy(meas_astrom)
-
+%{
+#include "ndarray/converter.h"
+#include "ndarray/converter/eigen.h"
+%}
+%include "ndarray.i"
 %include "lsst/pex/config.h"
 
 %shared_ptr(lsst::meas::astrom::MatchOptimisticBControl);
 
 %import "lsst/afw/table/tableLib.i"
 %import "lsst/afw/image/wcs.i"
 %import "lsst/afw/math/statistics.i"
+%import "lsst/afw/geom/geomLib.i"
 
 %include "lsst/meas/astrom/matchOptimisticB.h"
 
+%include "std_pair.i"
```

Comment:
If this could safely go higher (e.g. right near `%include "ndarray.i"` I think it would make the code a bit easier to read; we usually put standard and 3rd party includes together near the top when we can get away with it.

Another option worth considering is moving the new code into a separate .i file. There is a fair amount of it, so it may enhance readability.